content/blog/exposing-an-application-using-ingress-and-tls-termination-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,528 @@
+pod/ingress-nginx-controller-548768956f-8bz2q   1/1     Running   0          15m
+
+NAME                                         TYPE           CLUSTER-IP       EXTERNAL-IP    PORT(S)                      AGE
+service/ingress-nginx-controller             LoadBalancer   10.108.173.7     10.6.115.251   80:32734/TCP,443:32265/TCP   15m
+service/ingress-nginx-controller-admission   ClusterIP      10.108.100.150   <none>         443/TCP                      15m
+
+NAME                                       READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/ingress-nginx-controller   1/1     1            1           15m
+
+NAME                                                  DESIRED   CURRENT   READY   AGE
+replicaset.apps/ingress-nginx-controller-548768956f   1         1         1       15m
+```
+
+T﻿he service *ingress-nginx-controller* gets deployed as the service type of *LoadBalancer* with the *EXTERNAL-IP* assigned as *10.6.115.251*. This IP address will be used for setting up domain and subdomain name resolution.
+
+### Generate a self-signed certificate using cert-manager
+
+You can d﻿eploy cert-manager to the K8s cluster and generate a self-signed certificate by following up the blog post [Generating self-signed certificates using cert-manager](https://developer.hpe.com/blog/generating-self-signed-certificates-using-cert-manager-for-kubernetes-in-hpe-greenlake-for-private-cloud-entreprise/).
+
+H﻿ere is the deployed cert-manager to the namespace *cert-manager* in the cluster:
+
+```shell
+$ kubectl get all -n cert-manager
+NAME                                           READY   STATUS    RESTARTS   AGE
+pod/cert-manager-59fbb6655d-h7sqb              1/1     Running   0          18s
+pod/cert-manager-cainjector-69548575fb-7fqd2   1/1     Running   0          18s
+pod/cert-manager-webhook-57b78f476d-mp45s      1/1     Running   0          16s
+
+NAME                           TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+service/cert-manager           ClusterIP   10.107.221.97    <none>        9402/TCP   20s
+service/cert-manager-webhook   ClusterIP   10.104.243.185   <none>        443/TCP    19s
+
+NAME                                      READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/cert-manager              1/1     1            1           18s
+deployment.apps/cert-manager-cainjector   1/1     1            1           18s
+deployment.apps/cert-manager-webhook      1/1     1            1           17s
+
+NAME                                                 DESIRED   CURRENT   READY   AGE
+replicaset.apps/cert-manager-59fbb6655d              1         1         1       19s
+replicaset.apps/cert-manager-cainjector-69548575fb   1         1         1       19s
+replicaset.apps/cert-manager-webhook-57b78f476d      1         1         1       18s
+```
+
+Below is the deployed self-signed custom resource definition (CRD) *Issuer* in the namespace *nginx-apps* in which you want to generate certificate:
```

Comment:
Missing the word "the" just before "certificate":
"... you want to generate the certificate."